server/portal/apps/accounts/views.py

```diff
@@ -19,6 +21,28 @@
 logger = logging.getLogger(__name__)
 
 
+class LogoutView(DjangoLogoutView):
+    def dispatch(self, request, *args, **kwargs):
+        token = str(request.user.tapis_oauth.access_token)
+
+        logger.info('Attempting to revoke JWT token for user: ' + str(request.user))
```

Comment:
Should we use `request.user.username` instead of `str(request.user)`?

Reply:
Yep, totally forgot about that attribute.